src/Workspaces/Core/Portable/Utilities/Documentation/XmlDocumentationProvider.cs

```diff
@@ -23,6 +23,12 @@ public static XmlDocumentationProvider Create(byte[] xmlDocCommentBytes)
             return new ContentBasedXmlDocumentationProvider(xmlDocCommentBytes);
         }
 
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.FxCop.Rules.Security.Xml.SecurityXmlRules", "CA3053:UseXmlSecureResolver",
```

Comment:
I know you've merged this already, and it's totally fine to leave it, but one thing to consider would be to suppress it at the assembly level, since any instance of this violation in a portable assembly is going to be invalid.